ios/FluentUI/Core/ControlHostingContainer.swift

```diff
@@ -41,6 +41,12 @@ public class ControlHostingContainer: NSObject {
         self.updateRootView()
     }
 
+    deinit {
+        if let themeObserver = self.themeObserver {
+            NotificationCenter.default.removeObserver(themeObserver)
```

Comment:
i thought we don't need to do any removeobserver these days. do we? https://developer.apple.com/documentation/foundation/nsnotificationcenter/1413994-removeobserver

Reply:
Unfortunately that's only the case if you use `addObserver(_:selector:name:object:)`. If you observe via `addObserver(forName:object:queue:using:)` then you do still need to deregister once you're done observing.

Reply:
That said, I'm adding a little more logic in the notification handler, so I'm switching to `addObserver(_:selector:name:object:)` in both these cases.